config/v1/types_authentication.go

```diff
@@ -104,8 +105,14 @@ type AuthenticationStatus struct {
 	// The namespace for this config map is openshift-config-managed.
 	IntegratedOAuthMetadata ConfigMapNameReference `json:"integratedOAuthMetadata"`
 
-	// TODO if we add support for an in-cluster operator managed Keycloak instance
-	// KeycloakOAuthMetadata ConfigMapNameReference `json:"keycloakOAuthMetadata"`
+	// OIDCClients is where participating operators place the current OIDC client status
+	// for OIDC clients that can be customized by the cluster-admin.
+	//
+	// +listType=map
+	// +listMapKey=componentNamespace
+	// +listMapKey=componentName
+	// +kubebuilder:validation:MaxItems=20
+	OIDCClients []OIDCClientStatus `json:"oidcClients"`
```

Comment:
```suggestion
	// +listType=map
	// +listMapKey=componentNamespace
	// +listMapKey=componentName
	// +kubebuilder:validation:MaxItems=20
	// +openshift:enable:FeatureSets=CustomNoUpgrade;TechPreviewNoUpgrade
	OIDCClients []OIDCClientStatus `json:"oidcClients"`
```

somehow forgot/lost this